### ISSUE TITLE
Unable to install latest version (2.2.0rc1)?

### ISSUE THREAD

**Issue by sakinaljana** [tensorflow/text#251]
Hi TF-text team,

I'm trying to install the library using pip for the latest version (`2.2.0rc1`), but couldn't find any matching distribution. I'm using Python 3.7 and pip 20.0.2. The full errors is below

```
ERROR: Could not find a version that satisfies the requirement tensorflow-text==2.2.0rc1 (from versions: 0.1.0, 1.0.0b0, 1.0.0b1, 1.0.0b2, 1.15.0rc0, 1.15.1, 2.0.0rc0, 2.0.1, 2.1.0rc0)
ERROR: No matching distribution found for tensorflow-text==2.2.0rc1
```

From the [pypi](https://pypi.org/project/tensorflow-text/2.2.0rc1/) should be available, but why pip unable to find it. Any idea?

Thanks before

**Comment by broken** [comment]
It does, but doesn't support Windows at this time. Is that the OS you are trying to install from?

**Comment by sakinaljana** [comment]
@broken I'm using Mac, not Windows

**Comment by sakinaljana** [comment]
Anyway, i saw from here: https://pypi.org/project/tensorflow-text/2.2.0rc1/#files

The wheel name is contain mac_os_10_15. Is this mean we should Catalina (10.15) to install these? My current MacOS version is Mojave (10.14)

**Comment by broken** [comment]
Got it. It looks like our Mac build environment was erroneously updated. I'll look into this and see about building the proper versions.

**Comment by broken** [comment]
With it being Friday, I'm unable to get the build machines switched over, but should be able to early next week. Thanks again for catching this!

**Comment by sakinaljana** [comment]
Thanks @broken to following up this.

It should be ok right if install it directly using wheel?

**Comment by broken** [comment]
Yes; I think that should work. I just noticed that the wheel is of the right version, I think it is just getting labeled wrong...

**Comment by broken** [comment]
FYI; we just released 2.2.0rc2 which should be more widely available to previous versions of MacOS.

**Comment by gregbillock** [comment]
Looks like this is likely fixed -- if not, please re-open!
